src/components/TransactionDetails/index.tsx

```diff
@@ -31,23 +51,25 @@ type TransactionDetailsAmountItemProps = Omit<
   ComponentProps<typeof InlineTokenBalance>,
   "tokenAmount"
 > &
-  Pick<TransactionDetailsItemProps, "label"> & { tokenAmount?: string }
+  Omit<TransactionDetailsItemProps, "value"> & {
+    tokenAmount?: string
+  }
 
 export const TransactionDetailsAmountItem: FC<
   TransactionDetailsAmountItemProps
-> = ({ label, tokenAmount, ...restProps }) => {
-  const tokenBalanceTextColor = useColorModeValue("gray.700", "gray.300")
-
+> = ({ label, tokenAmount, isTotal, isSubTotal, ...restProps }) => {
```

Comment:
Instead of creating `isTotal` and `isSubTotal` boolean props here, I propose to just reate a `variant` property for `TransactionDetailsAmountItem`. Maybe something like this:

- `simple` - default one
<img width="616" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/40306834/444c8aee-3624-4666-8770-b0dd9c03a94b">

- `bold` (optionally it could be `secondary`):
<img width="628" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/40306834/6405aaa5-a56e-4529-875b-742e9a04327f">

- `highlight` (optionally it could be `primary`)
<img width="622" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/40306834/83b3b755-3ec9-4863-9cfb-46cb2a2d3c91">

if it's too much work, then we should at least change those names to `isBolded` and `isHighlighted` or something similar.

What do you think?

Reply:
Looking good 🔥